subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/corefeature/FileSystemWatchingPerformanceTest.groovy

```diff
@@ -29,8 +30,15 @@ class FileSystemWatchingPerformanceTest extends AbstractCrossVersionPerformanceT
 
     def setup() {
         runner.minimumBaseVersion = "6.7"
-        runner.targetVersions = ["6.8-20201002220441+0000"]
+        runner.targetVersions = ["6.8-20201007220043+0000"]
         runner.useToolingApi = true
+        if (OperatingSystem.current().windows) {
+            runner.warmUpRuns = 5
+            runner.runs = 20
+        } else {
+            runner.warmUpRuns = 10
+            runner.runs = 40
+        }
```

Comment:
I think we should always comment on these non-obvious things. Why here do we do something different on Windows?

Reply:
I added a comment.